cmd/libsnap-confine-private/bpf-insn.h

```diff
@@ -0,0 +1,311 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+/* imported from the Linux kernel, commit
+ * 77d34a4683b053108ecd466cc7c4193b45805528 (v5.13-11855-g77d34a4683b0) */
+
```

Comment:
Maybe import it under a subdirectory, e.g. `libsnap-confine-private/linux/bpf-insn.h`?

Reply:
Or `libsnap-confine-private/bpf/`? Also I could probably move the vendored headers to `libsnap-confine-private/bpf/vendor`, @mvo5 wdyt?

Reply:
Yeah, sounds good. This location means that I will have to find my own place to vendor "squashfuse" but that is ok, I think the dir layout suggested here makes sense :)

Reply:
> Or `libsnap-confine-private/bpf/`? Also I could probably move the vendored headers to `libsnap-confine-private/bpf/vendor`, @mvo5 wdyt?

Yes, either works, my main point was to have it separated from our headers.